### PR TITLE
fix: update code and method annotation

### PR DESCRIPTION
**Pull request by hakusai22**
1. Use is instead of == to check if a variable is None.
2. Method parameter annotation modification.
3. apple documentation link updates and additions.
4. word typo.

### CODE REVIEW

File: appstoreserverlibrary/models/ExtendRenewalDateRequest.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ class ExtendRenewalDateRequest:
     extendByDays: Optional[int] = attr.ib(default=None)
     """
     The number of days to extend the subscription renewal date.
-    
+    The number of days is a number from 1 to 90.
```
Comment (alexanderjordanbaker) [inline appstoreserverlibrary/models/ExtendRenewalDateRequest.py:20]: This doesn't follow the style of the rest of the library, where only the first sentence of a field's documentation is listed. Could you please remove this bit. Discussion about the style of documentation would probably be best in a separate issue

File: appstoreserverlibrary/models/TransactionHistoryRequest.py [hunk 2]
```diff
@@ -66,5 +66,5 @@ class TransactionHistoryRequest:
    
     revoked: Optional[bool] = attr.ib(default=None)
     """
-    An optional Boolean value that indicates whether the response includes only revoked transactions when the value is true, or contains only nonrevoked transactions when the value is false. By default, the request doesn't include this parameter.
+    An optional Boolean value that indicates whether the response includes only revoked transactions when the value is true, or contains only non-revoked transactions when the value is false. By default, the request doesn't include this parameter.
```
Comment (alexanderjordanbaker) [inline appstoreserverlibrary/models/TransactionHistoryRequest.py:69]: This differs from the official Apple documentation

File: appstoreserverlibrary/models/ExtendRenewalDateRequest.py (hunk 1 above)
Comment (hakusai22) [reply appstoreserverlibrary/models/ExtendRenewalDateRequest.py]: done

File: appstoreserverlibrary/models/TransactionHistoryRequest.py (hunk 2 above)
Comment (hakusai22) [reply appstoreserverlibrary/models/TransactionHistoryRequest.py]: done

**Review by alexanderjordanbaker — APPROVED** [review]
LGTM, thank you! I'll be bringing over the doc updates to the other three languages based on this. I think, besides the typos, we didn't originally list the link on a List of items, but I agree it makes the most sense to put the link in there.

**Pull request merged**